### ISSUE TITLE
Use HTML entities for feed titles

### ISSUE THREAD

**Issue by toastal** [jaspervdj/hakyll#574]
I have some titles with ampersands. The generated XML document becomes invalid from that point on, but with `&amp;` substituted for `&`, it's becomes valid again. The same could be said for `&lt;`, `&gt;`, etc.

**Comment by wisn** [comment]
@jaspervdj Are you accepting PR for this one?

**Comment by jaspervdj** [comment]
Yes I'd be happy to accept a PR for that.  Any PR for this should probably also have a test.
